src/themes/shared/borders.ts

```diff
@@ -3,4 +3,5 @@ export const borders = {
   '1px': '1px solid',
   '2px': '2px solid',
   '4px': '4px solid',
+  'inset 1px': 'inset solid 1px',
```

Comment:
I'm wondering if we really need this token here. Would you mind elaborating on the cases for adding it as token?

Reply:
@GutierrezGo we do not need that in tokens